vendor/k8s.io/kubernetes/staging/src/k8s.io/apiserver/pkg/server/genericapiserver.go

```diff
@@ -327,6 +328,14 @@ func (s preparedGenericAPIServer) Run(stopCh <-chan struct{}) error {
 
 		<-stopCh
 
+		// Set verbosity to at least 3 in order to log requests.
+		if !klog.V(3) && klog.V(1) {
```

Comment:
`!klog.V(3) && klog.V(1)` sounds logically equivalent to `klog.V(1)`

Reply:
no. it's 2 or 3.

Reply:
No, it's 1 or 2.